templates/repo/settings/options.tmpl

```diff
@@ -16,7 +16,7 @@
 					<label for="repo_name">{{.locale.Tr "repo.repo_name"}}</label>
 					<input id="repo_name" name="repo_name" value="{{.Repository.Name}}" data-repo-name="{{.Repository.Name}}" autofocus required>
 				</div>
-				<div class="inline field">
+				<div class="inline field{{if not (eq .Repository.LFSSize 0)}} tooltip{{end}}"{{if not (eq .Repository.LFSSize 0)}} data-content="{{.Repository.SizeDetailsString}}"{{end}}>
```

Comment:
```suggestion
				<div class="inline field{{if not (eq .Repository.Size 0)}} tooltip{{end}}"{{if not (eq .Repository.Size 0)}} data-content="{{.Repository.SizeDetailsString}}"{{end}}>
```

Reply:
why?

Reply:
Or should we always display `.Repository.SizeDetailsString` even if all of sizes are zero?

Reply:
> Or should we always display `.Repository.SizeDetailsString` even if all of sizes are zero?

Hmm, I think if `.Repository.Size` is zero, `.Repository.LFSSize` will always be also zero (Can't understand why push git lfs data without any git data ...). so it will't be shown also. now only show it when lfs size is not zero.

Reply:
> Isn't `.Repository.Size` means the total size includes Git Data size and LFS Data size.

Yes it does (at least currently) and so it seems in this PR

Reply:
@lunny I agree with @a1012112796 when there is no lfs there - there is nothing to "tooltip" about - no need to show the tool tip. So I would keep this like it was for now.